journalbeat/docs/index.asciidoc

```diff
@@ -13,6 +13,7 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 :github_repo_name: beats
 :discuss_forum: beats/{beatname_lc}
 :beat_default_index_prefix: {beatname_lc}
+:has_script_processor:
```

Comment:
I think this is missing ` :has_decode_csv_fields_processor:`.

Reply:
you're right! I forgot to push the commit where I've updated this.